### ISSUE TITLE
Training Wizards? How are they launched or enabled?

### ISSUE THREAD

**Issue by Duemellon** [d8ahazard/sd_dreambooth_extension#383]
Python 3.9.12 (main, Apr  4 2022, 05:22:27) [MSC v.1916 64 bit (AMD64)]
Commit hash: ****************************************
Installing requirements for Web UI
Skipping dreambooth installation.
Dreambooth revision is ****************************************
Diffusers version is *.*.*.
Torch version is *.*.*.
Torch vision version is *.*.*.
`^^ I do not know what this is referring to. I used Automatic1111 gradio gui to update to the latest ver of DB extension

**Have you read the [Readme](https://github.com/d8ahazard/sd_dreambooth_extension#readme)?** yes

**Have you completely restarted the stable-diffusion-webUI, not just reloaded the UI?** yes, a few times

**Have you updated Dreambooth to the latest revision?** yes

**Have you updated the Stable-Diffusion-WebUI to the latest version?** yes

**No, really. Please save us both some trouble and update the SD-WebUI and Extension and restart before posting this. Reply 'OK' Below to acknowledge that you did this.** done

**Describe the bug**
Why are Training Wizards unavailable? What is needed to turn them on?
I'm using Chrome. I disabled add blockers.
Did a "Find" on readme re: WIZARD and found nothing
I do not see any Wikis
Looked in discussions, searched for WIZARD but none had instructions
Is this broken? or am I misconfigured? or am I doing a sequence incorrectly?

![image](https://user-images.githubusercontent.com/113479041/205073753-5ddae1b4-8a97-40ce-b273-882187de07b6.png)

**Provide logs**

no crash

**Environment**

What OS? Windows 11

If Windows - WSL or native? Native 

What GPU are you using? AMD R7 5700

**Screenshots/Config**
If the issue is specific to an error while training, please provide a screenshot of training parameters or the db_config.json file from /models/dreambooth/MODELNAME/db_config.json

**Comment by dali99** [comment]
I guess it is a little incorrect to name them a wizard since they don't pop up any fancy gui that walks you through things.

They just set settings based on some of the other settings (i.e fill in your dataset folder and it will set learning rate and how many epochs automatically when you press the wizard button)

**Comment by Duemellon** [comment]
Are they supposed to give some feedback that it did such? or are they grayed out on mine? I'm not sure mine is working. 

I'm not able to train : L Trying to figure that out but the documentation is a bit behind so I'm lost.

**Comment by dali99** [comment]
No, no feedback other than the fields being updated to the numbers

**Comment by mykeehu** [comment]
Before using the button, create a model, then specify the folder of training images, and only then press the button, because it sets the parameters depending on the number of images. The button is not in a good place, it's true, but there's probably no better place for it for the moment.

**Comment by d8ahazard** [comment]
Wizards are not launched, per-se, it's just running some calculations. Buttons have been moved, and better output should be shown in the UI to indicate "things happened".
